qpc/report/tests_report_insights.py

```diff
@@ -111,38 +105,9 @@ def test_insights_report_as_json_report_id(self):
             args = Namespace(
                 scan_job_id=None, report_id="1", path=self.test_tar_gz_filename
             )
-            with redirect_stdout(report_out):
+            with self.assertLogs(level="INFO") as log:
                 nac.main(args)
-                self.assertEqual(
-                    report_out.getvalue().strip(), messages.REPORT_SUCCESSFULLY_WRITTEN
-                )
-
-    def test_insights_report_id_no_output_file(self):
-        """Testing retrieving insights with report id, without output file."""
-        report_out = StringIO()
-
-        get_report_url = get_server_location() + REPORT_URI + "1/insights/"
-        get_report_json_data = {
-            "id": 1,
-            "report_id": 1,
-            "hosts": {"00968d16-78b7-4bda-ab7d-668f3c0ef1ee": {"key": "value"}},
-        }
-        test_dict = {self.test_json_filename: get_report_json_data}
-        buffer_content = create_tar_buffer(test_dict)
-        with requests_mock.Mocker() as mocker:
-            mocker.get(
-                get_report_url,
-                status_code=200,
-                headers={"X-Server-Version": VERSION},
-                content=buffer_content,
-            )
-            nac = ReportInsightsCommand(SUBPARSER)
-            args = Namespace(scan_job_id=None, report_id="1", path=None)
-            with redirect_stdout(report_out):
-                nac.main(args)
-                self.assertIn(
-                    messages.REPORT_SUCCESSFULLY_WRITTEN, report_out.getvalue().strip()
-                )
+                self.assertIn(messages.REPORT_SUCCESSFULLY_WRITTEN, log.output[-1])
 
```

Comment:
watch out for this piece when rebasing. should have been removed in a fixup commit - I can help you later if you need

Reply:
ok!